includes/configuration_managers/class-newspack-ads-configuration-manager.php

```diff
@@ -32,6 +34,23 @@ public function is_configured() {
 		return class_exists( 'Newspack_Ads_Model' );
 	}
 
+	/**
+	 * Update custom targeting keys.
+	 *
+	 * @return string[]|WP_Error Created custom targeting keys or error if it fails.
+	 */
+	public function update_custom_targeting_keys() {
```

Comment:
Suggestion: we might need to do more setup chores on wizard load in the future. This could be a generic setup handler, with custom targeting key configuration as the single (for now at least) setup step. This could then be handled by `Newspack_Ads_Model`, so this plugin would not bother with the implementation details and would not have to be updated when new setup steps are added.